bin/generate-manifest.js

```diff
@@ -8,12 +8,9 @@ const manifest = Object.assign({
   'author': packageMeta.author,
   'version': packageMeta.version,
   'homepage_url': packageMeta.homepage,
-  'applications': {
-    'gecko': Object.assign({
-      'id': packageMeta.id
-    }, packageMeta.webextensionManifest.applications.gecko)
-  }
 }, packageMeta.webextensionManifest);
```

Comment:
Turns out that [the change I made earlier that changed the order of merging these objects](https://github.com/bwinton/SnoozeTabs/commit/4c4bbfd68d3aa1e924ec220e6a8e137b282a4f40#diff-ae48bfc07bcae523cc9f374db73f6a1cL7) just ended up clobbering the `applications.gecko.id` assignment altogether.

Reply:
Could we remove the id from `webExtensionManifest.applications.gecko` instead, if we know we want the packageMeta one?

Reply:
[Problem is that there isn't an id in there already](https://github.com/bwinton/SnoozeTabs/blob/master/package.json#L80). The only property in there is `update_url`, and that was clobbering the merged object

Reply:
Oh, cause they don't get merged… Dang. Okay, r=me.

Reply:
Yeah, `Object.assign` is a dumb flat merge ugh